tasks/config.js

```diff
@@ -0,0 +1,178 @@
+var websiteServer = null;
+
+var assignIn = require('lodash').assignIn;
+
+var CLIENT_TESTS_SETTINGS_BASE = {
+    port:            2000,
+    crossDomainPort: 2001,
+
+    scripts: [
+        { src: '/async.js', path: 'test/client/vendor/async.js' },
+        { src: '/hammerhead.js', path: 'node_modules/testcafe-hammerhead/lib/client/hammerhead.js' },
+        { src: '/core.js', path: 'lib/client/core/index.js' },
+        { src: '/ui.js', path: 'lib/client/ui/index.js' },
+        { src: '/automation.js', path: 'lib/client/automation/index.js' },
+        { src: '/legacy-runner.js', path: 'node_modules/testcafe-legacy-api/lib/client/index.js' },
+        { src: '/before-test.js', path: 'test/client/before-test.js' }
+    ],
+
+    configApp: require('../test/client/config-qunit-server-app')
+};
+var CLIENT_TESTS_PATH = 'test/client/fixtures';
+var CLIENT_TESTS_LEGACY_PATH = 'test/client/legacy-fixtures';
+
+var CLIENT_TESTS_SAUCELABS_SETTINGS = {
+    username:  process.env.SAUCE_USERNAME,
+    accessKey: process.env.SAUCE_ACCESS_KEY,
+    build:     process.env.TRAVIS_BUILD_ID || '',
+    tags:      [process.env.TRAVIS_BRANCH || 'master'],
+    name:      'testcafe client tests',
+    timeout:   720
+};
+
+var CLIENT_TESTS_DESKTOP_BROWSERS = [
+    {
+        platform:    'Windows 10',
+        browserName: 'microsoftedge'
+    },
+    {
+        platform:    'Windows 10',
+        browserName: 'chrome'
+    },
+    {
+        platform:    'Windows 10',
+        browserName: 'firefox'
+    },
+    {
+        platform:    'Windows 10',
+        browserName: 'internet explorer',
+        version:     '11.0'
+    },
+    {
+        platform:    'OS X 10.11',
+        browserName: 'safari',
+        version:     '9.0'
+    },
+    {
+        platform:    'OS X 10.11',
+        browserName: 'chrome'
+    },
+    {
+        platform:    'OS X 10.11',
+        browserName: 'firefox'
+    }
+];
+
+var CLIENT_TESTS_OLD_BROWSERS = [
+    {
+        platform:    'Windows 8',
+        browserName: 'internet explorer',
+        version:     '10.0'
+    },
+    {
+        platform:    'Windows 7',
+        browserName: 'internet explorer',
+        version:     '9.0'
+    }
+];
+
+var CLIENT_TESTS_MOBILE_BROWSERS = [
+    {
+        platform:    'Linux',
+        browserName: 'android',
+        version:     '5.1',
+        deviceName:  'Android Emulator'
+    },
+    {
+        platform:    'OS X 10.10',
+        browserName: 'iphone',
+        version:     '8.1',
+        deviceName:  'iPad Simulator'
+    },
+    {
+        platform:    'iOS',
+        browserName: 'Safari',
+        // NOTE: https://github.com/DevExpress/testcafe/issues/471
+        // problem with extra scroll reproduced only on saucelabs
+        // virtual machines with ios device emulators
+        version:     '9.3',
+        deviceName:  'iPhone 6 Plus'
+    }
+];
+
+
+function buildWebsite (mode, cb) {
```

Comment:
I have a feeling that this helper functions has nothing to do with config, maybe move it to `helpers.js` file?